### PR TITLE
Fix legacy problem

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

File: Commands.md [hunk 1]
```diff
@@ -35,13 +36,22 @@
...
+ - [`git rebase-patch`](#git-rebase-patch)
  - [`git release`](#git-release)
+ - [`git rename-tag`](#git-rename-tag)
  - [`git repl`](#git-repl)
  - [`git reset-file`](#git-reset-file)
  - [`git root`](#git-root)
  - [`git scp`](#git-scp)
  - [`git sed`](#git-sed)
  - [`git setup`](#git-setup)
+<<<<<<< HEAD
```
Comment (hemanth) [inline Commands.md]: 🌵

File: bin/git-fork [hunk 2]
```diff
@@ -20,7 +20,7 @@ read user
 # extract owner + project from repo url
 project=${url##*/}
 owner=${url%/$project}
-[[ $project == *.git ]] && project=${project%.git}
+project=${project%.git}
```
Comment (nicolaiskogheim) [inline bin/git-fork]: (Y)

**Comment by nicolaiskogheim** [comment]
Apart from the conflict markers, LGTM.

**Comment by spacewander** [comment]
Aha, I make a `pull -f` to fix it. 😄

**Pull request merged**